engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -60,11 +59,7 @@ trait CromwellApiService extends HttpInstrumentation {
   val engineRoutes = concat(
     path("engine" / Segment / "stats") { _ =>
       get {
-        onComplete(workflowManagerActor.ask(WorkflowManagerActor.EngineStatsCommand).mapTo[EngineStatsActor.EngineStats]) {
-          case Success(stats) => complete(ToResponseMarshallable(stats))
-          case Failure(e: TimeoutException) => e.failRequest(StatusCodes.ServiceUnavailable)
-          case Failure(_) => new RuntimeException("Unable to gather engine stats").failRequest(StatusCodes.InternalServerError)
-        }
+        completeResponse(StatusCodes.Forbidden, APIResponse.fail(new RuntimeException("The /stats endpoint is currently disabled.")), warnings = Seq.empty)
```

Comment:
Do you really need the overhead of creating an instantly-unpacked exception here? (ie is there not just a "use this message" version of the method?)

Also, it's not really a RuntimeException...

Reply:
This method works fine with a String but then returns `text/plain` rather than `application/json`. If you have specific suggestions for what this should be instead I'd be open to trying them.